pkg/apis/networking/v1beta1/register.go

```diff
@@ -42,14 +42,14 @@ var (
 // Adds the list of known types to the given scheme.
 func addKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(SchemeGroupVersion,
-		&v1beta1.AppliedToGroup{},
-		&v1beta1.AppliedToGroupPatch{},
-		&v1beta1.AppliedToGroupList{},
-		&v1beta1.AddressGroup{},
-		&v1beta1.AddressGroupPatch{},
-		&v1beta1.AddressGroupList{},
-		&v1beta1.NetworkPolicy{},
-		&v1beta1.NetworkPolicyList{},
+		&v1beta2.AppliedToGroup{},
```

Comment:
Since v1beta2 changes struct, we should continue serve legacy networking API with v1beta1 to make legacy clients compatible?

Reply:
+1, why is this file being changed?

Reply:
I was following networking/v1beta1/register.go where the known types were added as controlplane.v1beta1 types. In this case, we should still keep controlplae.v1beta1 types for `networking/v1beta1/register.go` and `controlplane/v1beta1/register.go`?

Reply:
Fixed